cmd/internal/cli/actions.go

```diff
@@ -59,26 +60,31 @@ func handleOCI(cmd *cobra.Command, u string) (string, error) {
 		return "", fmt.Errorf("failed to get SHA of %v: %v", u, err)
```

Comment:
The statement using `ImageSHA()` should be part of the else case of the if statement below since it is not used if the cache is disables